app/assets/stylesheets/styles/game.scss

```diff
@@ -40,13 +69,22 @@
 }
 
 #chessboard td.selected {
-  border: 2px solid #50bf79 !important;
+  -webkit-box-shadow: inset 0px 0px 4px 4px rgba(#b94343, 0.9);
+  box-shadow: inset 0px 0px 4px 4px rgba(#c41d1d, 0.88);
```

Comment:
Color literals like `rgba(#c41d1d, 0.88)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`0px` should be written without units as `0`